doc/nrf/gs_installing.rst

```diff
@@ -695,3 +564,152 @@ Define the required environment variables as follows, depending on your operatin
 You must also make sure that nrfjprog (part of the `nRF Command Line Tools`_) is installed and its path is added to the environment variables.
 The west command programs the kit by using nrfjprog by default.
 For more information on nrfjprog, see `Programming SoCs with nrfjprog`_.
+
+.. _gs_updating:
+
+Updating tools and repositories
+*******************************
+
+Remember to regularly check for updates to tools.
+Like any other Python package, the west tool is updated regularly.
+
+You might also want to switch to a newer release or check out the latest state of development.
+However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
+
+.. _west_update:
+
+Updating west
+=============
+
+To update west, run the following command:
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      .. parsed-literal::
+         :class: highlight
+
+         pip3 install -U west
+
+   .. group-tab:: Linux
+
+      .. parsed-literal::
+         :class: highlight
+
+         pip3 install --user -U west
+
+   .. group-tab:: macOS
+
+      .. parsed-literal::
+         :class: highlight
+
+         pip3 install -U west
+
+..
+
+Updating the repositories
+=========================
+
+To manage the ``nrf`` repository (the manifest repository), use Git.
+To make sure that you have the latest changes, run ``git fetch origin`` to :ref:`fetch the latest code <dm-wf-update-ncs>` from the `sdk-nrf`_ repository.
+Checking out a branch or tag in the ``nrf`` repository gives you a different version of the manifest file.
+Running ``west update`` will then update the project repositories to the state specified in this manifest file.
+
+.. include:: gs_installing.rst
+   :start-after: west-error-start
+   :end-before: west-error-end
+
+For example, to switch to release |release| of the |NCS|, enter the following commands in the ``ncs/nrf`` directory:
+
+.. parsed-literal::
+   :class: highlight
```

Comment:
You mean at the end of the line 623? Correct me if I'm wrong, but the `::` defaults to `.. code-block::` and it can also be easily mistaken as a typo by a zealous editor (not necessarily tech writer).